command/snapshot/inspect/snapshot_inspect.go

```diff
@@ -96,6 +263,9 @@ Usage: consul snapshot inspect [options] FILE
   To inspect the file "backup.snap":
 
     $ consul snapshot inspect backup.snap
-
+  
+  -enhance    enhance allows for more detailed information of the snapshot
+  such as message type, count, and size.
```

Comment:
Command line flags should be automatically added to the help text by `command/flags/usage.go` which we are calling above. So you don't need to add it here.